app/helpers/form_helper.rb

```diff
@@ -68,9 +68,6 @@ def us_states_territories
   private
 
   def validated_form_for(record, options = {}, &block)
```

Comment:
We can remove this helper now and just use `simple_form_for`, but I'll save that for a follow-on pull request, since it touches a lot of files.